context/app/static/js/helpers/init-trackers.js

```diff
@@ -1,41 +1,37 @@
-import { init as sentryInit } from '@sentry/react';
 import { TracingInstrumentation } from '@grafana/faro-web-tracing';
 import { LogLevel, getWebInstrumentations, initializeFaro } from '@grafana/faro-web-sdk';
+import { ReactIntegration } from '@grafana/faro-react';
 
 /**
- * Initialize Faro and Sentry tracking
+ * Initialize Faro trackers
  * This function should be used in the main `index.jsx` before the app is rendered.
  */
 function initTrackers() {
-  const release = `portal-ui-react@${PACKAGE_VERSION}`;
-  const enableTracking = ['prod', 'prod-stage'].includes(sentryEnv);
-
-  sentryInit({
-    dsn: sentryDsn,
-    environment: sentryEnv,
-    enabled: enableTracking,
-    release,
-  });
+  const version = `portal-ui-react@${PACKAGE_VERSION}`;
+  const environment = sentryEnv;
+  const paused = !['prod', 'prod-stage'].includes(environment);
 
   initializeFaro({
     url: 'https://faro-collector-prod-us-east-0.grafana.net/collect/77a0efade67edd876ae6c63ebb2d825c',
     app: {
       name: 'hubmap-data-portal',
-      version: release,
-      environment: sentryEnv,
-      paused: !enableTracking,
+      version,
+      environment,
+      paused,
     },
     instrumentations: [
       // Mandatory, overwriting the instrumentations array would cause the default instrumentations to be omitted
       ...getWebInstrumentations({
         captureConsole: true,
         capturePerformanceTimeline: true,
-        captureConsoleDisabledLevels: [LogLevel.INFO, LogLevel.DEBUG],
+        // Only capture errors and warnings in grafana
+        captureConsoleDisabledLevels: [LogLevel.INFO, LogLevel.DEBUG, LogLevel.LOG],
       }),
 
       // Initialization of the tracing package.
       // This packages is optional because it increases the bundle size noticeably. Only add it if you want tracing data.
       new TracingInstrumentation(),
+      new ReactIntegration(),
```

Comment:
I believe you previously mentioned not using the `ReactIntegration` what motivated the change?

Reply:
Now that we no longer have the `sentry` error boundary component, the `faro` error boundary component has become a relevant benefit to adding the react integration; previously, we did not immediately gain anything by adding this integration, as we were not immediately looking to start profiling performance (though we could add profiling now?), we do not use `react-router`, and already had an `ErrorBoundary` component that `faro`'s would have conflicted with.